fluent-langneg/src/accepted_languages.js

```diff
@@ -0,0 +1,10 @@
+
+export default function aceptedLanguages(string = '') {
+  if (typeof string !== 'string') {
+    throw new TypeError('Argument must be a string');
+  }
+  const tokens = string.split(',').map(t => t.trim());
+  return tokens.filter(t => t !== '').map(t => {
+    return t.split(';')[0];
+  });
```

Comment:
This is a nit, but there's no need to use `{}` here and an explicit `return`. Can you change this to a one-liner arrow function?